packages/amplify-provider-awscloudformation/src/utils/aws-logger.js

```diff
@@ -0,0 +1,39 @@
+const { logger } = require('amplify-cli-logger');
+
+export const fileLogger = file => (crumb, args) => error => {
+  if (error) {
+    logError({
+      mod: `${file}.${crumb}`,
+      args,
+      error,
+    });
+  } else {
+    logInfo({
+      mod: `${file}.${crumb}`,
+      args,
+    });
+  }
+};
+
+export function logStackEvents(events) {
```

Comment:
Can we stay consistent in the function syntax we use here? Either everything should be arrow or nothing should be. All function prototypes should be of the same format if possible when defining new functionality.